Themes/nightfall/nightfall.css

```diff
@@ -0,0 +1,2920 @@
+	margin-left: 10px;
+	padding: 5px 10px 5px 10px;
+	border-radius: 300px;
+	border-left: unset;
+}
+
+/*put in a hack on Followed Tag posts, bc I can't figure out how to make the username vertically align to middle
+.is_recommended.post_full .post_header .post_info .post_info_link:first-child, 
+.post .dockable_video_embed .post_header .post_info_tumblelog {
+	padding-top: 5px;
+	padding-bottom: 5px;
+}
+*/
+
+/*for some reason ask posts have a negative margin, so get rid of that*/
+.post_full.is_note .post_body {
+	margin-top: 0px;
+}
+
+/*fix spacing on headers in Search mode*/
+.post_brick.post .post_header {
+	line-height: unset;
+}
+
+.post_brick.post .post_header .tumblelog_avatar {
+	vertical-align: middle;
+}
+
+.post_brick.post .post_header .post-info-tumblelogs {
+	vertical-align: middle;
+	height: unset;
+	line-height: unset;
+}
+
+.post_brick.post .post_header .post-info-post-tumblelog, 
+.post_brick.post .post_header .post-info-post-tumblelog .worded-follow-button {
+	display: inline-block;
+}
+
+.post_brick.post .post_header .post-info-post-tumblelog .post-info-tumblelog {
+	vertical-align: middle;
+	display: inline-block;
+	max-width: 52%;
+	overflow: hidden;
+	text-overflow: ellipsis;
+}
+/*fix spacing on headers in Search mode done*/
+
+/*hack to make post header the right size for now
+.post .dockable_video_embed .embed_iframe {
+	top: 58px !important;
+}
+
+
+.post .dockable_video_embed.docked_post_underlay {
+	height: 272.875px !important;
+}
+*/
+
+/*special rules for dockable post headers*/
+.post.post_full .post.dockable_video_embed .post_header {
+	grid-template-columns: auto auto auto 1fr auto;
+	padding-top: 0px;
+	padding-bottom: 0px;
+	height: 44px;
+}
+
+/*hack to make post header the right size for now done*/
+/*fixing the post header's spacing and stuff, bc Tumblr doesn't know how to do anything done*/
+
+/*fix this weird type of radar post*/
+.radar .radar_post, 
+.radar.radar_type_regular .radar_content {
+	background: var(--post, #111);
+}
+
+.radar.radar_type_regular .header_flag {
+	background: #000;
+	border-top-color: var(--border, #444);
+	border-bottom-color: var(--border, #444);
+}
+
+.radar.radar_type_regular .radar_superglass {
+	background: unset;
+}
+
+.radar.radar_type_quote .radar_content, 
+.radar.radar_type_link .radar_content, 
+.radar.radar_type_converstaion .radar_content, 
+.radar.radar_type_regular .radar_content {
+	color: var(--text, #DDD);
+	padding-top: 20px;
+}
+
+.radar.radar_type_quote .radar_content:after, 
+.radar.radar_type_link .radar_content:after, 
+.radar.radar_type_converstaion .radar_content:after, 
+.radar.radar_type_regular .radar_content:after {
+	background: var(--border, #444);
+}
+/*fix this wierd type of radar post done*/
+
+/*make xkit audio slider a little more functional*/
+.audio-player {
+	padding-top: 0px;
+	padding-bottom: 25px;
+}
+
+.xkit-audio-plus-slider-container {
+	width: 455px;
+	left: 0px;
+	bottom: 0px;
+	border-radius: 0px;
```

Comment:
is this an issue or not? either way it shouldn't be a comment, it should be mentioned in the pull request description and/or logged in the issue tracker.